test/mock-game-contracts/mud.config.ts

```diff
@@ -114,3 +114,5 @@ export const configV2 = resolveStoreConfig({
     },
   },
 });
+
+export default config;
```

Comment:
OOC what's the reason for this change? just a style thing?

Reply:
oh this was just a temporary thing to be able to inspect individual fields of the config output better